packages/devtools/src/launcher.js

```diff
@@ -134,7 +134,7 @@ function launchBrowser (capabilities, product) {
 
     if (!executablePath) {
         throw new Error('Couldn\'t find executable for browser')
-    } else if (product === BROWSER_TYPE.firefox && !executablePath.toLowerCase().includes(CHANNEL_FIREFOX_NIGHTLY)) {
+    } else if (product === BROWSER_TYPE.firefox && executablePath !== 'firefox' && !executablePath.toLowerCase().includes(CHANNEL_FIREFOX_NIGHTLY)) {
```

Comment:
Some CI platforms disallow accessing binaries by a full path, like `/tmp/34rgew/firefox-nightly/firefox`.
The firefox is added to `PATH` and can be called with `firefox`.

To be able to run tests in CI I'd like to be able to set `binary` to just `firefox`, ex: `binary: process.env.CI ? 'firefox' : undefined,`